client/src/app/shared/shared-moderation/user-ban-modal.component.ts

```diff
@@ -56,6 +61,23 @@ export class UserBanModalComponent extends FormReactive implements OnInit {
           this.notifier.success(message)
 
           this.userBanned.emit(this.usersToBan)
+
+          if (mute) {
```

Comment:
Change `blockAccountByInstance` to accept an array and do the same logic than `banUsers` with `concatMap`

Then you'll just have to do `banUsers(...).pipe(switchMap(this.muteAccounts...))`